### ISSUE TITLE
Great plug-in, hoping to support defining the specific time of each conversation.

### ISSUE THREAD

**Issue by eryajf** [holubj/obsidian-dialogue-plugin#5]
Sometimes I need to record the specific time of each conversation, hoping to support it.

**Comment by dleeftink** [comment]
+1, just as Dataview support would be amazing! 

``` dialogue
left: speaker a
right: speaker b

< [ 12:01:00 :: transcript a ]
> [ 12:01:04 :: transcript b ]
```

This can possibly exposed to Dataview by adding `speakers` as inline tags/links and timing as JS/luxon/momentjs datetime objects per line.

**Comment by 4r7if3x** [comment]
I agree, that's the only thing it lacks in comparison with the other [Chat View](https://github.com/adifyr/obsidian-chat-view) plugin, yet I like your syntax and UI better.
